java-core/solid/fruit-shop-tests.md

```diff
@@ -61,8 +61,14 @@ public void getReport_Ok() {
     Assert.assertEquals(expected, actual);
 }
 
-@After
+@AfterEach
 public void afterEachTest() {
     Storage.storage.clear();
 }
 ```  
+#### Test your strategy and all handlers separately from each other and from services
+<b>Unit testing is isolated testing. Don't forget this.</b> <br>
+Don't test your strategy in service (e.g. `FruitService`) where you use it. It's enough to create map with only one 
+handler type (e.g. `Balance`) and test it works.
+
+Test all your handlers separately. Create test for every single handler and test as many scenarios as possible.
```

Comment:
Looks more clear
```suggestion
#### Unit testing is **isolated** testing
Keep your strategy, handler, and service tests separate from each other. Each of them needs a separate test class with its corresponding test cases.
Don't test your strategy in service (e.g. `FruitService`). It's enough to create a map with only one handler (e.g. `Balance`).
```

Reply:
Good point. Just one note: the bold text doesn't work with handlers. I'd prefer an underline